Stock-signalling.py

```diff
@@ -0,0 +1,57 @@
+import requests

```

Comment:
Good work Suwayda. This is a clear and concise code for monitoring specific stock prices and notifying the user when the price drops by at least £0.25.
Your comments are to the point and help me understand your thought process. 
However, I don't see a code for getting the 7 day average, this is something that you could add in future versions of the code, it will result in a better stock monitoring app, as you will have historical data and be better able to track drastic changes in the stock price. This is important as it will help inform the user on when to purchase more stocks.

Otherwise, you followed the syntax in naming variables and functions, your code is very readable, I would be able to understand and continue from where you had stopped if we were handing over. Easy to debug or go back and make changes, because things are clearly named.